_content/booklets/mettabhavana_anandajoti.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Mettabhāvanā (Universal version) Friendliness Meditation"
+authors:
+  - anandajoti
+external_url: "https://ancient-buddhist-texts.net/Texts-and-Translations/Short-Pieces/Mettabhavana-2.htm"
```

Comment:
Wrong title and link. The pdf on Drive is the Sri Lankan version: https://ancient-buddhist-texts.net/Texts-and-Translations/Short-Pieces/Mettabhavana-1.htm